esphome/components/ds2484/ds2484.cpp

```diff
@@ -0,0 +1,209 @@
+#include "ds2484.h"
+
+namespace esphome {
+namespace ds2484 {
+static const char *const TAG = "ds2484.onewire";
+
+void DS2484OneWireBus::setup() {
+  ESP_LOGCONFIG(TAG, "Setting up DS2484 1-wire bus...");
+  this->reset_device();
+  this->search();
+}
+
+void DS2484OneWireBus::dump_config() {
+  ESP_LOGCONFIG(TAG, "DS2484 1-wire bus:");
+  this->dump_devices_(TAG);
+}
+
+bool DS2484OneWireBus::read_status_(uint8_t *status) {
+  for (int retry_nr = 0; retry_nr < 10; retry_nr++) {
```

Comment:
No need to use an `int` to count to ten. 🙂
```suggestion
  for (uint8_t retry_nr = 0; retry_nr < 10; retry_nr++) {
```